src/plugins/badges.jl

```diff
@@ -34,3 +34,18 @@ function badges(::ColPracBadge)
         "https://github.com/SciML/ColPrac",
     )
 end
+
+"""
+    PkgEvalBadge()
+
+Adds a [`PkgEval`](https://github.com/JuliaCI/PkgEval.jl) badge to the [`Readme`](@ref) file.
```

Comment:
Since the PkgEval.jl link doesn't explain mention the badge, perhaps we should link to https://github.com/JuliaCI/NanosoldierReports#badges too somehow?

Perhaps either:
```suggestion
Adds a [`PkgEval`](https://github.com/JuliaCI/PkgEval.jl) [badge](https://github.com/JuliaCI/NanosoldierReports#badges) to the [`Readme`](@ref) file.
```
or maybe better:
```suggestion
Adds a [`PkgEval` badge](https://github.com/JuliaCI/NanosoldierReports#pkgeval-reports) to the [`Readme`](@ref) file.
```